docs/rules-engine/how-to-use/integration.md

```diff
@@ -40,6 +40,18 @@ import { RulesEngineModule } from '@o3r/rules-engine';
 export class AppModule {}
 ```
 
+#### Action executor Modules
+
+The result of the Rules Engine is a set of actions apply to the runtime of the application.
+To not make the Rules Engine requiring all the modules it is probably apply actions to, the import of the different modules needs to be handle by the application integrating the Rules Engines.
```

Comment:
```suggestion
The result of the Rules Engine is a set of actions applied at runtime on the application.
To not make the Rules Engine require all the modules on which it can possibly apply actions, the import of the different modules needs to be handled by the application integrating the Rules Engines.
```
not sure I understand the second sentence though :)

Reply:
@fpaul-1A : Seems correct with these changes:
Not to make (To be or not to be), or to avoid the issue:

"To avoid the dependency of the rules engine on all the modules it can possibly apply actions on, the import of the different modules shall be the handled by the application ..."